src/test/java/hudson/plugins/emailext/ExtendedEmailPublisherDescriptorTest.java

```diff
@@ -83,13 +83,14 @@ public class ExtendedEmailPublisherDescriptorTest {
     public JenkinsRule j = new JenkinsRule();
 
     private void assertTitlePage(HtmlPage page) {
-        VersionNumber jenkinsVersion = Jenkins.getVersion();
-        VersionNumber newManageJenkinsVersion = new VersionNumber("2.395");
-        String expectedTitle = "System [Jenkins]";
-        if (jenkinsVersion.isOlderThan(newManageJenkinsVersion)) {
-            expectedTitle = "Configure System [Jenkins]";
-        }
-        assertEquals("Should be at the Configure System page", expectedTitle, page.getTitleText());
+        // oldTitleFormat can be removed once our baseline is high enough
+        String oldTitleFormat = "System [Jenkins]";
```

Comment:
I would just not check the Jenkins suffix

Reply:
Good idea - updated.